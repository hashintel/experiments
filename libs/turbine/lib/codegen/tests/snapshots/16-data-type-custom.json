[
  {
    "$schema": "https://blockprotocol.org/types/modules/graph/0.3/schema/data-type",
    "kind": "dataType",
    "$id": "https://localhost:3000/@alice/types/data-type/custom/v/1",
    "title": "Text",
    "description": "An ordered sequence of characters",
    "type": "string"
  }
]
