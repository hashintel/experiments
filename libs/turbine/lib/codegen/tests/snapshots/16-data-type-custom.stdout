localhost_3000/alice/data/custom.rs

compile_error ! ("custom data types are not yet supported, see [#355](https://github.com/blockprotocol/blockprotocol/pull/355) for the official RFC") ;
